pramen/core/src/test/scala/za/co/absa/pramen/core/tests/runner/task/TaskRunnerBaseSuite.scala

```diff
@@ -336,10 +336,12 @@ class TaskRunnerBaseSuite extends AnyWordSpec with SparkTestBase with TextCompar
         """[ {
           |  "a" : "B",
           |  "b" : 2,
+          |  "INFO_DATE" : "2022-02-18",
```

Comment:
And this is the test for the new logic - the information date is now available after a jobs has run, even before it is saved.